src/uti.ts

```diff
@@ -115,7 +115,18 @@ export const formatTable = (
 
   return `${tableHeader}\n${tableSeparator}\n${gameRows}\n`;
 };
-
+export const formatStatsTable = (stats: Stats): string => {
+  const tableHeader = `| Rapid | Blitz | Bullet |`;
```

Comment:
Maybe we can add some icons on the headers, as we have in the stats table?
Blitz could be ⚡  for example.

Reply:
Okay